src/site/antora/modules/ROOT/pages/manual/flowtracing.adoc

```diff
@@ -131,83 +116,145 @@ public void someMethod() {
 ----
 <1> The `throwing()` method logs exceptions that are thrown and not caught.
 
-== Differences in flow tracing methods
+[#aop]
+=== Aspect-oriented programming
+
+Logging has been a notorious example for demonstrating https://en.wikipedia.org/wiki/Aspect-oriented_programming[aspect-oriented programming] (AOP).
+For instance, using AOP, you can inject logging statements to methods that match a particular footprint, e.g., all public methods in `com.mycompany` package.
+With a couple of lines of AOP instructions, you can log input and output of all matching functions.
+Flow tracing methods fits like a glove to this AOP use case.
+You can see a demonstration of this in https://github.com/apache/logging-log4j-samples/tree/main/log4j-samples-aspectj[the `log4j-samples-aspectj` project] demonstrating how you can implement this use case using Log4j API flow methods and Spring Boot AspectJ support.
+
+[#impl]
+== Implementation
+
+This section explains how flow tracing is implemented by different logging implementations.
 
-Flow tracing methods have specific markers assigned and logs with a level of `TRACE`.
-It's also noteworthy that all messages begin with "event".
+[#impl-log4j]
+=== Log4j Core
 
-The table below shows the methods and their unique features.
+Log4j Core, the reference implementation of Log4j API, implements the flow tracing methods such that
 
-[cols="3,3,3", options="header"]
+* <<impl-log4j-log-events,It generates specialized log events>>
+* <<impl-log4j-config,Its behaviour is configurable>>
+
+[#impl-log4j-log-events]
+==== Log events
+
+Log4j Core implements the flow tracing methods such that the generated log events are decorated to accommodate any need to selectively handle them:
```

Comment:
Although I am not strongly opinionated about this, I would prefer to view this section as section that applies to all Log4j API implementations.

Although this is not part of the contract, all known implementation translate flow tracing calls into `log` calls with the given parameters.

Reply:
I will kindly pass this Piotr. Though thanks for the suggestion.